tox.ini

```diff
@@ -5,7 +5,7 @@ skip_missing_interpreters = True
 envlist = py310,py39,py38,py37,flake8,black,twine-check,mypy,isort,cz,pylint
 
 [testenv]
-passenv = GITLAB_IMAGE GITLAB_TAG PY_COLORS NO_COLOR FORCE_COLOR DOCKER_HOST PWD
+passenv = GITHUB_ACTIONS GITLAB_IMAGE GITLAB_TAG PY_COLORS NO_COLOR FORCE_COLOR DOCKER_HOST PWD
```

Comment:
Seems like we need it.

https://github.com/utgwkk/pytest-github-actions-annotate-failures/blob/55bba59b58955c26cb657dee39b8551909277b88/pytest_github_actions_annotate_failures/plugin.py#L25-L28